config/settings.js

```diff
@@ -51,12 +60,31 @@ window.spinnakerSettings = {
         region: cfDefaultRegion
       },
     }
+    titan: {
+      defaults: {
+        account: 'titantest',
+        region: 'us-east-1'
+      },
+    },
+    kubernetes: {
+      defaults: {
+        account: 'my-kubernetes-account',
+        namespace: 'default'
+      },
+    }
+  },
+  whatsNew: {
+    gistId: '32526cd608db3d811b38',
+    fileName: 'news.md',
   },
-  authEnabled: false,
+  authEnabled: process.env.AUTH === 'enabled',
   feature: {
     pipelines: true,
     notifications: false,
-    rebakeControlEnabled: true,
+    fastProperty: true,
```

Comment:
These 3 values should all be `false`, and `rebakeControlEnabled` should be `true`.